apps/docs/pages/integrations/angular.mdx

```diff
@@ -34,28 +34,38 @@ To get started make sure to install the packages using your favorite package man
 
 <Callout>Make sure, that you have set skipLibCheck to true in your tsconfig</Callout>
 
+### Create your config
+
+To use A/BBY you need to create your config first. You can do this by creating a file called `abby.config.ts` in your `root` folder. This file will be used to configure your project.
+
+```ts
+// abby.config.ts
+import { defineConfig } from "@tryabby/angular";
+
+export default defineConfig({
+  projectId: "<YOUR_PROJECT_ID>",
+  currentEnvironment: environment.production ? "production" : "development",
+  environments: ["production", "development"],
+  tests: {
+    test: { variants: ["A", "B"] },
+    footer: { variants: ["dark", "orange", "green"] },
+    // ... your tests
+  },
+  flags: { darkMode: "Boolean", newFeature: "Boolean" },
```

Comment:
where and how can I define my default values? :thinking:

Reply:
Good catch, it's missing in the angular docs